src/features/filter.js

```diff
@@ -0,0 +1,13 @@
+import { useSelector } from 'react-redux';
+
+const handleSearch = (event) => {
+  const countries = useSelector((state) => state.myred);
+  const value = event.target.value.toLowerCase();
+  let result = [];
+  console.log(value);
```

Comment:
- [ ] Optional suggestion, kindly remove the `console.log` from the code that has been pushed.

Reply:
Noted, Thank you!